### DISCUSSION TITLE
Unsupported zigbee Moes Zigbee lightswitch ZTS-EU

### DISCUSSION THREAD

**Discussion by adxnop** [arendst/Tasmota#12204] [Ideas]
- - Mercy please for make it support 🙏🙏🙏🙏

**Comment by sfromis** [comment]
It is most probably already supported. Lots of devices does not have the "confirmed" indicator, but that only means that no one has reported it to be working. Nearly all simpler Zigbee devices is expected to work without someone having to do something.
https://zigbee.blakadder.com/Moes_ZTS-EU3-W.html

**Reply by adxnop** [reply]
Hi sfromis
Thanks for replied. Tasmota is able to pair it but when I send command to it it is shown "Missing zigbee 'Send', 'Write', 'Report' or 'Response'"

When I try to on/off it manually it is shown like this
tele/tasmota_E304CB/LIVINGROOM1LIGHT32/SENSOR = {"ZbReceived":{"LIVINGROOM1LIGHT32":{"Device":"0x1D84","Name":"LIVINGROOM1LIGHT32","EF00/040E":0,"Endpoint":1,"LinkQuality":81}}}

I have no idea how to send this via Zbsend

**Reply by sfromis** [reply]
Cluster EF00 tends to be Tuya, and they have a bad habit of grossly breaking Zigbee standards. AFAIK it is still possible to control, with a bit of extra work with the codes, but I'm not into the details there.

Personally, I'd skip any Zigbee device wanting to be in the Tuya ecosystem. Bad citizens.....
